File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/checkin/CheckInEvent.kt
package com.hhp227.concafe.presentation.main.checkin

sealed interface CheckInEvent {
    data class NavigateToCafe(val id: String) : CheckInEvent

    data class NavigateToCast(val id: String) : CheckInEvent

    data class NavigateToReviewEdit(val cafeId: String) : CheckInEvent

    data object NavigateToSignIn : CheckInEvent

    data object OpenLocationSettings : CheckInEvent

    data class ShowMessage(val message: String) : CheckInEvent
}

File: iosApp/iosApp/Presentation/Main/CheckIn/CheckInEvent.swift
//
//  CheckInEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/06.
//

import Foundation

enum CheckInEvent {
    case navigateToCafe(id: String)
    case navigateToCast(id: String)
    case navigateToReviewEdit(cafeId: String)
    case navigateToMap
    case navigateToSignIn
    case openLocationSettings
}
